operations/mimir-mixin/dashboards/queries.libsonnet

```
@@ -167,6 +167,157 @@ local filename = 'mimir-queries.json';
         { fieldConfig+: { defaults+: { unit: 'short' } } },
       )
     )
+    .addRowIf(
+      $._config.show_ingest_storage_panels,
+      ($.row('Ingester (ingest storage: strong consistency)'))
+      .addPanel(
+        $.timeseriesPanel('Requests with strong consistency') +
```

[nit] WDYT about being more specific about the "strong consistency"?

```suggestion
        $.timeseriesPanel('Requests with strong read consistency / sec') +
```

_If you agree, same comment applies in other places._

---

Sounds good, fixing.